README.md

```diff
@@ -90,13 +82,13 @@ sudo apt-get install postgresql postgresql-contrib
 Clone this repository to your desired folder:
 
 cd my-folder
-git clone git@github.com:David63011/vet-clinic-database.git
+git clone github.com/david63011/school-library.git
 
 ### Install
 
```

Comment:
**[ OPTIONAL]** 
- Since this is a group project, the Author section in your ReadMe should hvae the contact details of both ream members. I strongly advise you add this information